action/bucket.go

```diff
@@ -15,7 +14,7 @@ func MakeBucket(remote string) (err error) {
 		return
 	}
 	// Init bucket
-	if err = helper.PutBucket(bucketName, contexts.Zone); err != nil {
+	if err = contexts.Storage.SetupBucket(bucketName, contexts.Zone); err != nil {
```

Comment:
this should be `contexts.Storage.PutBucket`, `SetupBucket` did not send request

Reply:
Fixed.